### PR TITLE
Add kafka topic for expunge

### PR DESCRIPTION
**Pull request by maulikjs**

### CODE REVIEW

**Review by accorvin — APPROVED** [review]

File: kafka/secrets.enc.yaml [hunk 1]
```diff
@@ -1,171 +1,172 @@
 dh_kafka_git_branch: production
 dh_kafka_git_repo: https://github.com/AICoE/internal-data-hub.git
```
Comment (anishasthana) [inline kafka/secrets.enc.yaml:2]: Not part of the changes, but this needs to be updated right?

Comment (maulikjs) [reply kafka/secrets.enc.yaml:2]: Yeah makes sense
I will raise another MR for it

**Pull request merged**